test/helpers/data/aggregated-data-helper.js

```diff
@@ -144,17 +135,24 @@ var addOffenderManager = function (inserts) {
     })
     .then(function (ids) {
       inserts.push({table: 'workload_owner', id: ids[0]})
-      return knex('workload').returning('id').insert(Object.assign({}, defaultWorkload, {workload_owner_id: ids[0]}))
+      var workloads = [
+        Object.assign({}, defaultWorkload, {workload_owner_id: ids[0]}),
+        Object.assign({}, defaultWorkload, {workload_owner_id: ids[0]})
+      ]
+      return knex('workload').returning('id').insert(workloads)
     })
     .then(function (ids) {
-      inserts.push({table: 'workload', id: ids[0]})
-
-      var workloadReports = inserts.filter((item) => item.table === 'workload_report')
+      ids.forEach((id) => {
+        inserts.push({table: 'workload', id: id})
+      })
 
+      return knex('workload_report').select('id').orderBy('effective_from', 'desc')
+    }).then(function (workloadReports) {
```

Comment:
It is just the id but if we were to name it `workloadReportIds` then in order to access the first id we would still have to reference `workloadReportIds[0].id`. This is because the object actually does represent a workload report object, even though, in this instance, we are only returning the ID component. After a conversation about it, we've agreed to leave it as it is.